src/swap/index.ts

```diff
@@ -8,6 +8,7 @@ export type {
   BuildSwapTransaction,
   BuildSwapTransactionResponse,
   Fee,
+  FromTo,
```

Comment:
Why are exporting this to all developers, it feels very vague and not sure I see the usage for them.

Reply:
removed